config/environments/development.rb

```diff
@@ -33,7 +33,7 @@
   # metadata (jsonb) fields using the @> operator. If the right operand contains a dot separated value
   # (example: "test.com") Rails was interpreting this as a table/column and this was breaking the query
   # AH 3/10/16
-  config.active_record.disable_implicit_join_references = true
```

Comment:
See the comment I had left here. Was something changed in Rails or the project that prevents this from breaking our metadata searches?